### PR TITLE
Integrate Grizzly 4.0.1

### PR DESCRIPTION
**Pull request by pzygielo**
- https://github.com/eclipse-ee4j/glassfish/pull/24612
```
18:20:25  + mvn -B -e clean
18:20:26  [INFO] Error stacktraces are turned on.
18:20:26  [INFO] Scanning for projects...
18:20:28  [ERROR] [ERROR] Some problems were encountered while processing the POMs:
18:20:28  [ERROR] Non-resolvable import POM: org.glassfish.grizzly:grizzly-bom:pom:4.0.1-M1 was not found in https://repo.eclipse.org/content/repositories/maven_central/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of eclipse.maven.central.mirror has elapsed or updates are forced @ org.glassfish.main:nucleus-parent:7.0.10-SNAPSHOT, /home/jenkins/agent/workspace/_test-using-jenkinsfile_PR-24612/nucleus/parent/pom.xml, line 404, column 25
```

### CODE REVIEW

**Comment by arjantijms** [comment]
Oh wow, I totally missed that. Thanks! :)

File: runtests.sh [hunk 1]
```diff
@@ -27,13 +27,15 @@ catch() {
...
   mvn clean -N org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy \
+  -Pstaging \
   -Dartifact=org.glassfish.main.distributions:glassfish:${GF_VERSION}:zip \
   -Dmdep.stripVersion=true \
   -DoutputDirectory=${WORKSPACE}/bundles
 }
 
 install_jacoco() {
   mvn -N org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy \
+  -Pstaging \
```
Comment (dmatej) [inline runtests.sh:38]: This is redundant, jacoco is not in staging, just in central, also this command doesn't use poms  ;)

File: runtests.sh [hunk 2]
```diff
@@ -27,13 +27,15 @@ catch() {
 
 install_glassfish() {
   mvn clean -N org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy \
+  -Pstaging \
```
Comment (dmatej) [inline runtests.sh:30]: Same here

**Comment by pzygielo** [comment]
Thanks for checking.
Review comments implemented (changes undone).

**Comment by arjantijms** [comment]
4.0.1-M1 can be updated to 4.0.1 (same code, M1 works).

**Review by dmatej — APPROVED** [review]

**Review by avpinchuk — APPROVED** [review]

**Pull request merged**